### PR TITLE
Add OVMSLauncher

### PR DESCRIPTION
**Pull request by vinnamkim**
### Summary

- Ticket no. 112210
- Implement OVMSLauncher

### How to test
I added some tests for this new feature.

### Checklist

- [x] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [x] I have added the description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md).​
- [ ] I have updated the [documentation](https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly

### License

- [x] I submit _my code changes_ under the same [MIT License](https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: src/datumaro/plugins/ovms_plugin/launcher.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+        results = self._client.predict(
+            inputs={self._input_key: inputs},
+            model_name=self.model_name,
+            model_version=self.model_version,
+            timeout=self.timeout,
+        )
+
+        # If there is only one output key,
+        # it returns `np.ndarray`` rather than `Dict[str, np.ndarray]`.
+        # Please see ovmsclient.tfs_compat.grpc.responses.GrpcPredictResponse
```
Comment (sooahleex) [inline src/datumaro/plugins/ovms_plugin/launcher.py:160]: It seems good to add [GrpcClient.predict](https://github.com/openvinotoolkit/model_server/blob/releases/2022/3/client/python/ovmsclient/lib/docs/grpc_client.md#method-predict) in this comment or add link of this in documentation for this feature.

File: src/datumaro/plugins/ovms_plugin/samples/face_detection.py
Comment (sooahleex) [inline src/datumaro/plugins/ovms_plugin/samples/face_detection.py:1]: If this sample is related to [Face Blur Pipeline Demo with OVMS](https://docs.openvino.ai/2023.0/ovms_demo_face_blur_pipeline.html), I think it seems good to add this link too.

**Review by sooahleex — DISMISSED** [review]
Looks good to me.

File: src/datumaro/plugins/ovms_plugin/samples/face_detection.py
Comment (vinnamkim) [reply src/datumaro/plugins/ovms_plugin/samples/face_detection.py:1]: No, it is not about the face blur, but about just face detection, https://docs.openvino.ai/2023.0/ovms_demo_face_detection.html. However, this use case looks good to me make a Jupyter notebook example for `OVMSLauncher`. Thanks for letting me know.

File: src/datumaro/plugins/ovms_plugin/launcher.py (hunk 1 above)
Comment (vinnamkim) [reply src/datumaro/plugins/ovms_plugin/launcher.py:160]: Updated in https://github.com/openvinotoolkit/datumaro/pull/1056/commits/afd0b4007e96c26de2d43e3d5df1f4c2a78d1799

**Review by sooahleex — APPROVED** [review]

**Review by wonjuleee — APPROVED** [review]

**Pull request merged**